modules/coreaudio/coreaudio.c

```diff
@@ -105,14 +117,17 @@ CFStringRef coreaudio_get_device_uid(const char *name)
 		warning("AudioObjectGetPropertyDataSize"
 			" (kAudioHardwarePropertyDevices) failed: %i\n",
 			status);
+		err = 1;
```

Comment:
please use a Posix error code here. for example `ENODEV` or `ENOENT`.
(or a different one to describe the error).